src/wp-includes/class-wp-theme-json.php

```diff
@@ -626,6 +631,7 @@ protected static function sanitize( $input, $valid_block_names, $valid_element_n
 		foreach ( $valid_element_names as $element ) {
 			$schema_styles_elements[ $element ] = $styles_non_top_level;
 
+			// Note: array_key_exists() should be replaced with an isset() check once WordPress drops support for PHP 5.6.
```

Comment:
Same here for using `See: URL.` to link off for additional details/context.

There are several more throughout the changes here, but I'm going to stop commenting on each one inline.